service/src/main/kotlin/fi/espoo/evaka/invoicing/service/generator/VoucherValueDecisionGeneratorV2.kt

```diff
@@ -106,7 +106,7 @@ fun generateVoucherValueDecisionsDrafts(
             minDate = minDate,
         )
 
-    val newDrafts = voucherBases.map { it.toVoucherValueDecision() }
+    val newDrafts = voucherBases.mapNotNull { it.toVoucherValueDecision() }
```

Comment:
what's the case where draft would have null end date? is it ok to throw it away here?